projects/ngrx.io/content/examples/ngrx-start/src/main.ts

```diff
@@ -1,8 +1,5 @@
-import './polyfills';
```

Comment:
I think the changes to the examples can be reverted?
Or is this needed?

Reply:
@timdeschryver ...While I was working the example in the docs I had noticed the example starter in stackblitz was throwing errors and wouldn't compile. I was just removing lines the editor was complaining about. Being my first ever contribution to any open source project I will do whatever you guys suggest.

Reply:
I don't seem to be getting the error. Could you try again?

I think we can revert this for now, and see if it still occurs when we upgrade the deps to use the latest versions.